tests/test_keeper_chief.py

```diff
@@ -108,3 +114,50 @@ def test_check_hat(self, mcd: DssDeployment, keeper: ChiefKeeper, guy_address: A
         assert self.spell.eta() != 0
 
 
+    def test_check_eta_receipt(
+        self, mcd: DssDeployment,
+        keeper: ChiefKeeper,
+        simpledb: SimpleDatabase,
+        our_address: Address
+    ):
+        print_out("test_check_eta_receipt")
+
+        # clear out anything that came before
+        keeper.check_hat()
+        keeper.check_eta()
+
+        # Give 1000 MKR to our_address
+        amount = Wad.from_number(5000)
+        mint_mkr(mcd.mkr, our_address, amount)
+        assert mcd.mkr.balance_of(our_address) == amount
+
+        # Lock MKR in DS-Chief
+        assert mcd.mkr.approve(
+            mcd.ds_chief.address
+        ).transact(from_address=our_address)
+        assert mcd.ds_chief.lock(amount).transact(from_address=our_address)
+
+        # Deploy spell
+        spell = DSSBadSpell.deploy(mcd.web3)
+
+        # Vote 5000 mkr on the spell
+        assert mcd.ds_chief.vote_yays(
+            [spell.address.address]
+        ).transact(from_address=our_address)
+
+        keeper.check_hat()
+
+        block = mcd.web3.eth.blockNumber
+        simpledb.update_db_etas(block)
+
+        hat = mcd.ds_chief.get_hat()
+
+        etas = keeper.database.db.get(doc_id=3)['upcoming_etas']
+        verify([hat.address], etas, 1)
+
+        keeper.check_eta()
+
+        # Confirm that the spell was casted and that the database was updated
+        assert DSSBadSpell(mcd.web3, Address(hat)).done() == False
```

Comment:
Could you add a little more context either in this comment or as a comment near the test method header that points out the modification in `cast()`? 

Someone with familiarity of how `casts` are supposed to function in a properly written `DSSSpell`  may be quick to assume that this comment, and thus test, must be a false positive.

Reply:
good point, I'll add more context.

Reply:
added more context